FileIO/readMeshFromFile.cpp

```diff
@@ -44,8 +44,12 @@ MeshLib::Mesh* readMeshFromFile(const std::string &file_name)
 {
 #ifdef USE_PETSC
 	NodePartitionedMeshReader read_pmesh(PETSC_COMM_WORLD);
+
+	std::size_t pos = file_name.find_last_of(".");
+	std::string file_name_ext = file_name.substr(pos, file_name.size()-1);
```

Comment:
Did not check that. Just replaced the current implementation with the function. Thanks.